src/sentry/tsdb/snuba.py

```diff
@@ -164,6 +199,10 @@ def get_data(
         `group_on_time`: whether to add a GROUP BY clause on the 'time' field.
         `group_on_model`: whether to add a GROUP BY clause on the primary model.
         """
+        # XXX: to counteract the hack in project_key_stats.py
```

Comment:
The hack I'm referring to: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_key_stats.py#L36-L38

Reply:
This throws if the key is a string not containing a number. Are you sure keys are always numbers?

Reply:
Yea. The key based models are only ever called from `project_key_stats` so feel pretty safe about that.